### DISCUSSION TITLE
Lycoris models

### DISCUSSION THREAD

**Discussion by Enferlain** [lllyasviel/stable-diffusion-webui-forge#1417] [Ideas]
I was trying to test a boft model I trained, and it seems like forge doesn't pick it up at all. If not supported atm, can we add support for them, or does it depend on something else?

![image](https://github.com/user-attachments/assets/c2da1053-0b8a-4d06-b957-3dcddb556e70)

**Comment by lllyasviel** [comment]
the files are moved to https://github.com/lllyasviel/stable-diffusion-webui-forge/tree/main/packages_3rdparty/webui_lora_collection
But parsing them is WIP, you can give an example LoRA and I will use it as an example to activate these files

**Reply by Enferlain** [reply]
cool, got 2 types

[DiagOFT](https://drive.google.com/file/d/1ZA-QzCHjDQitpNFfdnJ3voMWzPuencoy/view?usp=drive_link)
[BOFT](https://drive.google.com/file/d/1tveOqViM9nDyJW7iDfajua1tboxkDpj_/view?usp=drive_link)

https://lora-inspector.rocker.boo/

don't have files for the others
